tests/www/test_views.py

```diff
@@ -122,31 +122,39 @@ def local_context(self):
         return result
 
 
-class TestBase(unittest.TestCase):
-    @classmethod
-    @dont_initialize_flask_app_submodules(
-        skip_all_except=[
-            "init_appbuilder",
-            "init_appbuilder_views",
-            "init_flash_views",
-            "init_jinja_globals",
-        ]
-    )
-    def setUpClass(cls):
+class TestBase:
+    @pytest.fixture(scope="class")
```

Comment:
Making this session scoped will I think make a big difference.

Reply:
And if you make this a fixture in `tests/www/conftest.py` then it can be used by `tests/www/test_app.py` etc too (I have a change where I'd like to use this in, for instance.)

Reply:
But can we? `minimal_app` is mutated in a lot of test preparations, and making the fixture module-scoped would make those side effects leak out to other tests. For example, I don’t think `init_api_connexion` can be called multiple times on the same app. `TestAirflowBaseViews` also sets `app.dag_bag` which would affect subsequent tests using the same app.

I agree with moving it to `conftest.py` though. I’m keeping things as local as possible for now, and will move it when I start working on other files in `tests/www`.

Reply:
I tried making `minimal_app` module-scope and `TestDagRunModelView::test_clear_dag_runs_action` fails. And interestingly, it also does not speed up the tests 🤔